homeassistant/components/zwave_js/api.py

```diff
@@ -436,6 +443,78 @@ def filename_is_present_if_logging_to_file(obj: dict) -> dict:
     return obj
 
 
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/start_listening_logs",
+        vol.Required(ENTRY_ID): str,
+    }
+)
+async def websocket_start_listening_logs(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict
+) -> None:
+    """Start listening to logging messages from the server."""
+    entry_id = msg[ENTRY_ID]
+    client = hass.data[DOMAIN][entry_id][DATA_CLIENT]
+    driver = client.driver
+
+    @callback
+    def async_cleanup() -> None:
+        """Remove signal listeners."""
+        asyncio.run_coroutine_threadsafe(
```

Comment:
We're in async context so we can use `hass.async_create_task`. If we need to await the coroutine to stop the log before unsubscribing the unsubs we should make `async_cleanup` a coroutine function and wrap that in a task that we schedule from a callback.

Reply:
I think it's ok to not await the coroutine